content/blog/2019/testthat-2-3-0/index.Rmarkdown

```diff
@@ -22,12 +21,12 @@ knitr::opts_chunk$set(
 )
 ```
 
-We're pumped to announce that [testthat 2.3.0](http://testthat.r-lib.org) is now available on CRAN! 
-testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily. 
-testthat is the most popular unit-testing package for R, and is used by over 4,000 CRAN and Bioconductor packages. 
-You can learn more about unit testing at <https://r-pkgs.org/tests.html>. 
+We're pumped to announce that [testthat 2.3.0](http://testthat.r-lib.org) is now available on CRAN!
+testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily.
+testthat is the most popular unit-testing package for R, and is used by over 4,000 CRAN and Bioconductor packages.
+You can learn more about unit testing at <https://r-pkgs.org/tests.html>.
 
-(We didn't write a blog post about testthat 2.2.0 because it only introduced a single, experimental, new feature: `verify_output()`. 
+(We didn't write a blog post about testthat 2.2.0 because it only introduced a single, experimental, new feature: `verify_output()`.
 It has now matured to the point we think you should also try it out, so it's discussed below.)
```

Comment:
Personally, I am not a fan of stand-alone parentheticals, _but_, given we don't have a great format for an aside, I'm not sure what the best alternative would be here.